people/htcai.md

```diff
@@ -0,0 +1 @@
+Greetings my name is Haitao Cai. I introduced my self [here](https://github.com/cognoma/cognoma/issues/2#issue-163984722).
```

Comment:
@htcai, your introduciton link should be changed to https://github.com/cognoma/cognoma/issues/2#issuecomment-230884763

Would you like to add another commit to your `addMyName` branch with this change?